gateway-api-spec.yaml

```diff
@@ -911,6 +913,8 @@ components:
           $ref: "#/components/schemas/NetworkIdentifier"
         at_state_identifier:
           $ref: "#/components/schemas/PartialLedgerStateIdentifier"
```

Comment:
We should update the documentation for both of these endpoints to explain that you can get an ASC order by adding a "from state identifier" - so that it appears in ReDocly here: :)

* https://docs.radixdlt.com/main/apis/api-specification.html
* https://redocly.github.io/redoc/?url=https://raw.githubusercontent.com/radixdlt/radixdlt-network-gateway/1.1.6/gateway-api-spec.yaml#tag/Account/paths/~1account~1transactions/post

Reply:
Agree. However I wanted to align on something first. We use this `PartialLedgerStateIdentifier` and its `description` field. Right now we'd have to either drop `description` from `#/components/schemas/PartialLedgerStateIdentifier` or use different type for `from_ledger_state` or maybe do something else like "override description of from ledger state" but is it something we want to do?

Reply:
Hmm good point. I think we can actually add a description here, which is technically invalid, but will win.

Or, to do it in a valid way, I think we might be able to use an allOf to add a new description over the top of the base type?

Reply:
Used simpler trick. I've just changed description to:

Optional. Allows a client to request a response referencing an earlier (`at_*` properties) or later (`from_*` properties) ledger state.

I guess this should be enough?

Reply:
Sure, works for me :P